src/openrct2/title/TitleSequence.cpp

```diff
@@ -202,22 +196,19 @@ bool TitleSequenceAddPark(TitleSequence* seq, const utf8* path, const utf8* name
 {
     // Get new save index
     size_t index = SIZE_MAX;
-    for (size_t i = 0; i < seq->NumSaves; i++)
+    for (size_t i = 0; i < seq->NumSaves(); i++)
     {
-        if (String::Equals(seq->Saves[i], path, true))
+        if (String::Equals(seq->Saves[i].c_str(), path, true))
         {
             index = i;
             break;
         }
     }
     if (index == SIZE_MAX)
     {
-        seq->Saves = Memory::ReallocateArray(seq->Saves, seq->NumSaves + 1);
-        Guard::Assert(seq->Saves != nullptr, GUARD_LINE);
-        index = seq->NumSaves;
-        seq->NumSaves++;
+        index = seq->NumSaves();
     }
-    seq->Saves[index] = String::Duplicate(name);
+    seq->Saves.insert(seq->Saves.begin() + index, std::string(name));
```

Comment:
Not sure this is doing the same thing as before.

Previously it searched for an index to replace or increased the vector by one and used the last one if the vector was full. Now it is always adding a new element at index, which is alright for `SIZE_MAX`, but does add a duplicate when `index = i`, does it not?

(This is in commit https://github.com/OpenRCT2/OpenRCT2/pull/12815/commits/946a8fd52fd8899e373f1c63f11bd802014151c5)